docs/spark-procedures.md

```diff
@@ -421,12 +421,18 @@ Existing data files are added to the Iceberg table's metadata and can be read us
 
 To leave the original table intact while testing, use [`snapshot`](#snapshot) to create new temporary table that shares source data files and schema.
 
+Migrate will create a backup table with name [`table__BACKUP__`]. If you feel confident that the migration succeeded 
+feel free to drop the backup table. You can also explicitly pass  [`drop_backup => true`] to drop the backup table after 
+migration finishes.
+
 #### Usage
 
-| Argument Name | Required? | Type | Description |
-|---------------|-----------|------|-------------|
-| `table`       | ✔️  | string | Name of the table to migrate |
-| `properties`  | ️   | map<string, string> | Properties for the new Iceberg table |
+| Argument Name | Required? | Type | Description                                                                          |
+|---------------|---------|------|--------------------------------------------------------------------------------------|
+| `table`       | ✔️ | string | Name of the table to migrate                                                         |
+| `properties`  | ️ | map<string, string> | Properties for the new Iceberg table                                                 |
```

Comment:
Can you revert the changes to the other lines? I don't think these were updated so there is no need to modify them.

Reply:
Done. fixed in the latest commit.